doc/source/scripts/gen_topologyparser_attrs.py

```diff
@@ -88,46 +81,27 @@ def _keys(parser: TopologyReaderBase) -> tuple[str, str]:
                 key = label = f
             return (key, label)
 
-        def _description(
```

Comment:
Don't rewrite code that you don't need to touch.

(applies to many more lines)